scripts.py

```diff
@@ -1870,34 +1870,43 @@ def do_run(self):
             concurrently = ''
         else:
             concurrently = 'CONCURRENTLY'
-        # Initialize database
+
+        # Initialize the default database session. We can't use this
+        # session to run the VACUUM commands, because it wraps
+        # everything in a big transaction, and VACUUM can't be
+        # executed within a transaction block. But we will use it at
+        # the end, to recalculate the sizes of lanes.
         db = self._db
+
+        url = Configuration.database_url()
+        engine = create_engine(url, isolation_level="AUTOCOMMIT")
+        engine.autocommit = True
+        a = time.time()
+        engine.execute("VACUUM (VERBOSE, ANALYZE)")
```

Comment:
Is this verbose for debugging purposes? Does making it verbose affect performance?

Reply:
It is for debugging purposes, and it doesn't affect performance -- it prints a few lines for every affected table and index.